docker-compose.staging.yaml

```diff
@@ -4,8 +4,10 @@ services:
 
   app:
     image: usedgoodreads/app:staging
-    ports:
-      - "8080:5000"
 
   worker:
     image: usedgoodreads/app:staging
+
+  nginx:
+    volumes:
+      - ./nginx/certs:/certs
```

Comment:
Staging also gets deployed right now, but to port 8080 - can we use the same certificates as on production, or why are we using the one from local development here?

Reply:
That is true. Using now same certs in staging with be12887658bf4dde05f3e45f86b5b0c9024c023c